### DISCUSSION TITLE
Glossary reviewer

### DISCUSSION THREAD

**Discussion by tm-yegorova** [open-metadata/OpenMetadata#11557] [General]
Hi, I've crashed something and I no can't choose reviewer and owner of the glossary absolutly. However I have a lot of users.

I don't understand how to configurate users to see them as reviewers. Week ago I could add them to reviewers simply. An then I've crashed something (I don't know)/

![glossary](https://github.com/open-metadata/OpenMetadata/assets/49267105/dc56a7be-6add-48a7-a395-c5d13b754783)
![users](https://github.com/open-metadata/OpenMetadata/assets/49267105/aa8c1b40-1801-4c8a-a903-5ad7417ef69f)

Should I give users  special role  or add them to the same team?

Now when I come as admin, I can't add reviewer and owner: list is empty.
When I come as another user and make new glossary I can add myself as owner and can't add reviewer: list is empty.

Fnd another question. When I add somebody as a glossary reviewer previously, he didn't get any special previlegies to make review. What does it mean to be a reviewer? Which functional is?

**Comment by harshach** [comment]
@tm-yegorova please go to settings -> OpenMetadat -> search -> reIndex all

**Reply by tm-yegorova** [reply]
Thank you ver mach. It works!! :) 
Could you help with reviewers roles? Which role should I give for reviewer? Now reviewers don't have special opportunities in Glossary.
